scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/JobData.java

```diff
@@ -262,6 +267,11 @@ InternalJob toInternalJob() {
         internalJob.setMaxNumberOfExecution(getMaxNumberOfExecution());
         internalJob.setOnTaskError(OnTaskError.getInstance(this.onTaskErrorString));
         internalJob.setScheduledTimeForRemoval(getScheduledTimeForRemoval());
+        try {
+            internalJob.setResultMap(SerializationUtil.deserializeVariableMap(getResultMap()));
+        } catch (IOException | ClassNotFoundException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)